staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1057,6 +1057,8 @@ func (r *Request) DoRaw(ctx context.Context) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	metrics.RequestLatency.Observe(ctx, r.verb, r.finalURLTemplate(), time.Since(start))
```

Comment:
Do you plan to add a latency per unit size?

Reply:
I want to give the users/admins the opportunity to estimate bandwidth (bits per second), that will allow to easily identify devices that rate limit traffic, per example to 10 mpbs ;)